src/components/CompareResults/SubtestsResults/SubtestsResultsMain.tsx

```diff
@@ -81,6 +68,34 @@ function SubtestsResultsMain({ view }: SubtestsResultsMainProps) {
     updateRawSearchParams(rawSearchParams);
   };
 
+  if (!results.length) {
+    return (
+      <React.Fragment>
```

Comment:
no need to change, but as a reminder current versions of React accept to use `<>` instead of `React.Fragment`, I find it less noisy and faster to type, but it's really a stylistic nit so this can also stay that way.